src/main/webapp/index.html

```diff
@@ -82,19 +90,27 @@
             printWindow.document.close();
         }
 
-        function lookup_student() {
+        function lookup_student(readonly) {
             $('#student_id').blur();
             var id = document.getElementById("student_id");
             var section = document.getElementById("section_student_info");
             var info = document.getElementById("student_info");
+            if (readonly =="true"){
```

Comment:
Can you please do consistent formatting?  Please insert spaces where appropriate. Also do this for Java code.